mappings/upheno_custom_mapping_head_100.sssom.tsv

```diff
@@ -0,0 +1,101 @@
+subject_id	subject_label	subject_source	predicate_id	object_id	object_label	object_source	mapping_justification
```

Comment:
1. use converter.compress_or_standardize() from the curies package for the subject_id and object_id
2. manually curate a yaml file with relevant metadata in this directory: https://github.com/monarch-initiative/monarch-mapping-commons/tree/main/metadata
3. Add the instructions on how to generate this file in https://github.com/monarch-initiative/monarch-mapping-commons/blob/main/monarch_mapping_commons.Makefile